src/Illuminate/Http/FileHelpers.php

```diff
@@ -46,6 +48,6 @@ public function hashName($path = null)
             $path = rtrim($path, '/').'/';
         }
 
-        return $path.md5_file($this->getRealPath()).'.'.$this->guessExtension();
+        return $path.Uuid::uuid4()->toString().'.'.$this->guessExtension();
```

Comment:
I think a UUID is more likely to be unique than random_bytes, right? UUID's strength is uniqueness, random_bytes's strength is cryptographically secure randomness.